### ISSUE TITLE
Pivot table isGroupField: false is not working 

### ISSUE THREAD

**Issue by eppsaksh** [free-jqgrid/jqGrid#136]
Hi Oleg,

When I try to use isGroupField: false on x dimension columns, by default it shows me grouping on that column.
I don't want grouping on x dim-columns, Which parameter is applicable to avoid grouping when needed.

Thanks,
eppsaksh

**Comment by OlegKi** [comment]
Sorry, but I can't follow you. How looks the options of jqPivot which you use? How loos the input data? Which version of free jqGrid you use?

I don't understand which scenario you try to implement and why you use `isGroupField` option. Probably you need just use `grouping: false` option of jqGrid (include the option to the options of jqGrid specified by second parameter of `jqPivot`)?

**Comment by eppsaksh** [comment]
Hi Oleg,

I am using jqGrid  4.5.2 - version, my project cannot upgrade it as we are already freeze with most of requirement on same version.
We want to develop pivot, so using only grid.pivot.js.
 
We just want to display data in pivot format where Group collapse/expand feature not needed.
 Like below
![image](https://cloud.githubusercontent.com/assets/13976360/9545324/11f82622-4da6-11e5-8a7a-e941431aaefb.png)

Thanks
eppsaksh

**Comment by OlegKi** [comment]
Sorry, but I asked you many questions. You answered only on one question. If I understand you correctly that you use `grid.pivot.js` from some unknown version of free jqGrid and all other parts you get from jqGrid 4.5.2. It seems to me a possible scenario, but you don't posted **how you use `jqPivot`** (which parameters you used)?

The method `jqPivot` have 4 parameters (see [the line](https://github.com/free-jqgrid/jqGrid/blob/v4.9.2/js/grid.pivot.js#L730) of code). The 3-d parameter `gridOpt` allows to specify the options of jqGrid *which append or overwrite the options* generated during by internal `pivotSetup` method. I wrote you before that I suggest you just add `grouping: false` to the list of jqGrid options (the 3-d parameter `gridOpt`). It should fix your problem. If it will have some problems, then **you should post the code which shows how you use `jqPivot` currently**.

**Comment by eppsaksh** [comment]
Hi Oleg,

I am really sorry for inconvenience. 
Q. How looks the options of jqPivot which you use?
A. 
Code as below
  var grid = $("#pivotGridv1");
	    grid.jqGrid("jqPivot",
	    		'../listEmpMaster.srvc',
	        {    	
	            xDimension: [
	                {
			    isGroupField: false,
	                    width: 100,
	                    dataName: 'glname',
	                    label: 'GEN Ledger',
	    	            grouping: false
	                },
	                {
	                    isGroupField: false,
	                    width: 100,
	                    dataName: 'slname',
	                    label: 'SUB Ledger',
	                    sortable:true,
	    	            grouping: false
	                }
	            ],
	            yDimension: [
			{    dataName: 'month',
			    width: 5,
			    label: 'Month',
			    rowTotals: true,   
			    rowTotalsText: "Month Total"					
			}
	            ],
	            aggregates: [
	                {	member: "amount",    aggregator: "sum",label: "{0}"}
	            ],
	           
	            grouping: false,
	            useColSpanStyle: true,
	            rowTotals: true, 
	            colTotals: true	          
	        },
	    // grid options
	        {
	        	  width: 1000,
	        	    height: 'auto',
	        	    rowNum: 5,
	        	    pager: "#pivotGridPager",
	        	    caption: "No Grouping",
	        	    sorting :true	        	        	 
	    },{ reader : "gridRecords" }).jqGrid("gridResize");

Q. How loos the input data? 
A. Data is from database server call 
![image](https://cloud.githubusercontent.com/assets/13976360/9546507/d97925c0-4db0-11e5-86f9-4ce006be802f.png)

Q. Which version of free jqGrid you use?
A. Already answered. File grid.pivot.js is copied from below path
https://github.com/free-jqgrid/jqGrid/blob/master/js/grid.pivot.js

Thanks,
Akshada

**Comment by OlegKi** [comment]
Sorry Akshada, but I have to repeat that jqPivot have 4 parameters (see [the line](https://github.com/free-jqgrid/jqGrid/blob/v4.9.2/js/grid.pivot.js#L730) of code):
```JavaScript
jqPivot: function (data, pivotOpt, gridOpt, ajaxOpt) {...}
```
If we reformat the code, which you posted, to make it more readable, it will be like the following:
```JavaScript
var grid = $("#pivotGridv1");
grid.jqGrid("jqPivot",
    '../listEmpMaster.srvc',
    {
        xDimension: [
            {
                isGroupField: false,
                width: 100,
                dataName: 'glname',
                label: 'GEN Ledger',
                grouping: false
            },
            {
                isGroupField: false,
                width: 100,
                dataName: 'slname',
                label: 'SUB Ledger',
                sortable: true,
                grouping: false
            }
        ],
        yDimension: [
            {
                dataName: 'month',
                width: 5,
                label: 'Month',
                rowTotals: true,
                rowTotalsText: "Month Total"
            }
        ],
        aggregates: [
            { member: "amount", aggregator: "sum", label: "{0}" }
        ],
        grouping: false,
        useColSpanStyle: true,
        rowTotals: true,
        colTotals: true
    },
    // grid options
    {
        width: 1000,
        height: 'auto',
        rowNum: 5,
        pager: "#pivotGridPager",
        caption: "No Grouping",
        sorting: true
    },
    { reader: "gridRecords" }
).jqGrid("gridResize");
```
So you used the first parameter `data` as `'../listEmpMaster.srvc'`, the second `pivotOpt` parameter as
```JavaScript
{
    xDimension: [
        {
            isGroupField: false,
            width: 100,
            dataName: 'glname',
            label: 'GEN Ledger',
            grouping: false
        },
        {
            isGroupField: false,
            width: 100,
            dataName: 'slname',
            label: 'SUB Ledger',
            sortable: true,
            grouping: false
        }
    ],
    yDimension: [
        {
            dataName: 'month',
            width: 5,
            label: 'Month',
            rowTotals: true,
            rowTotalsText: "Month Total"
        }
    ],
    aggregates: [
        { member: "amount", aggregator: "sum", label: "{0}" }
    ],
    grouping: false,
    useColSpanStyle: true,
    rowTotals: true,
    colTotals: true
}
```
the 3-d parameter `gridOpt` as
```JavaScript
{
    width: 1000,
    height: 'auto',
    rowNum: 5,
    pager: "#pivotGridPager",
    caption: "No Grouping",
    sorting: true
}
```
and the 4-th `ajaxOpt` parameter as `{ reader: "gridRecords" }`.

In other words you placed `grouping: false` many times in absolutely wrong place. You have to use `grouping` as the part of `gridOpt` parameter, for example as:
```JavaScript
{
    grouping: false, // !!! HERE !!!
    gridview: true, // It's recommended to improve the performance
    width: 1000,
    height: 'auto',
    rowNum: 5,
    pager: "#pivotGridPager",
    caption: "No Grouping",
    sorting: true
}
```
You should of cause remove `isGroupField: false`, `grouping: false` and `sortable: true` used in many places of the second `pivotOpt` parameter.

**Comment by eppsaksh** [comment]
Hi Oleg,

Thanks for solution. Its working good now.

Regards,
eppsaksh

**Comment by OlegKi** [comment]
You are welcome! I close the issue.
